visidata/undo.py

```diff
@@ -95,6 +95,15 @@ def _undo():
             c.name = name
     vd.addUndo(_undo)
 
+@VisiData.api
+def addUndoReload(vd, rows, cols):
+    oldrows = rows
+    oldcolumns = [copy(c) for c in cols]
```

Comment:
I think just `oldcolumns = cols` is fine, like `rows`, since reload assigns a new list to both rows and columns.